mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.js

```diff
@@ -75,6 +75,7 @@ export class ExperimentRunsTableMultiColumnView2 extends React.Component {
     filter: true,
     suppressMenu: true,
     suppressMovable: true,
+    maintainColumnOrder: true,
```

Comment:
What is required for? Maybe I am missing something but I can't find any reference for `maintainColumnOrder`.

Reply:
I saw it mentioned [here](https://www.ag-grid.com/react-data-grid/column-updating-definitions/#maintain-column-order) in the documentation of the Ag Grid. But I think I interpreted it incorrectly. I was trying to deal with the issue in which a re-checked column would appear at the end of the column list instead of in the place where it was before. See an example below where version and user have been unchecked and rechecked and ended up at the end of the list.

![image](https://user-images.githubusercontent.com/9256667/135903528-881dc788-362b-433f-a8ad-85acc25ff828.png)

In any case, i'll remove the `maintainColumnOrder` property here.

Reply:
I think this is a good change to do, so that when we check and uncheck columns the order is maintained. Feel free to add it back or we can add it another PR. :)
Didn't know this API existed. Thank you for pointing to the API.